app/views/comments/_comment_form.html.haml

```diff
@@ -7,5 +7,3 @@
   %p
     = link_to "How to comment", @application.comment_url
     on this application (from #{@application.authority.full_name}).
-    - if @application.on_notice_to
```

Comment:
It looks like you're shifting this from the wrong place. Have a look at when this notice would be displayed, only when the authority isn't contactable and the comment form isn't displayed. In that case there would be no comments either, so the notice isn't actually being pushed down the page—it's kinda in the perfect place with the comment instructions.

We want to keep this one here and move the notice that appears with the form. Let me know if you need a hand finding that.